magento/magento2
# Reverting javascript translation refactoring which broke javascript translation

### Description

This reverts commit 459ce25867fe931bdac856ec2cd275731e29640e.

The commit broke some javascript translations, because after javascript translation initialisation was refactored into its own file, the translation dictionary was not initialised early enough.

### Fixed Issues (if relevant)

1. magento/magento2#19705: Javascript translation issue on modal buttons when removing items from product compare page

### Manual testing scenarios (*)

1. Create a translation dictionary file for a storefront language
2. Translate the phrase "Cancel" in the dictionary file
3. Clear cache, deploy static content, ...
4. Click on the remove product link in minicart
5. Observe wether the cancel-button of the confirmation-popup is translated

### Contribution checklist (*)
